src/oci-umount.c

```diff
@@ -327,7 +326,7 @@ static int prestart(const char *rootfs,
 
 		real_path = realpath(line, NULL);
 		if (!real_path) {
-			pr_pinfo("Failed to canonicalize path [%s]. Skipping.", line);
+			pr_pinfo("Failed to canonicalize path [%s]: %m. Skipping.", line);
```

Comment:
This is C code. Where is err variable here and err.Error() interface. Am I missing something.